organization/settings.py

```diff
@@ -729,3 +734,18 @@
 ARTICLE_KEYWORDS = ['', ]
 
 DATA_UPLOAD_MAX_NUMBER_FIELDS = 10000
+
+# OAUTH2
+OAUTH2_IRCAM = True
+
+OAUTH_SERVER_BASEURL = 'https://auth.ircam.fr/'
+USER_SERVER_BASEURL = 'https://auth.ircam.fr/'
+#OAUTH_SERVER_BASEURL = 'http://my-oauth2-server.dev:8000'
+#USER_SERVER_BASEURL = 'http://localhost:8130'
```

Comment:
You can remove the comments here.
To allow local testing, these settings should be moved to environment.

Reply:
done